# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gtkextra/fix_autogen.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./autogen.sh.old	Thu Jul 18 20:55:51 2002
+++ ./autogen.sh	Sun Feb  2 22:41:49 2003
@@ -50,7 +50,8 @@
 if ( aclocal --version ) </dev/null > /dev/null 2>&1; then
 	rm -f aclocal.m4
 	echo -n "Building aclocal.m4... "
-	${ACLOCAL}
+	${ACLOCAL} -I /opt/gnome2/share/aclocal
+	${ACLOCAL} -I /opt/gnome24/share/aclocal
 	echo "done."
 else
 	echo "aclocal not found -- aborting `basename $0`"
